src/librbd/mirror/snapshot/GroupUnlinkPeerRequest.cc

```diff
@@ -53,25 +51,23 @@ void GroupUnlinkPeerRequest<I>::unlink_peer() {
       }
 
       if (ns->mirror_peer_uuids.empty() ||
-	(ns->mirror_peer_uuids.count(peer) != 0 &&
```

Comment:
Why is this check being dropped?  It wasn't part of my https://github.com/ceph/ceph/pull/65524#discussion_r2351466411 suggestion.